tests/test_twodstats.py

```diff
@@ -136,3 +190,4 @@ def generate_starlist(n_samples=500):
 if __name__ == '__main__':
     test_twodstats()
     # yaml test is in test_simple
```

Comment:
Indeed, perhaps this comment is tell us that once you break up `test_single_image` into two, the second half should be moved to this file.